bigchaindb/common/schema/__init__.py

```diff
@@ -18,7 +18,7 @@ def validate_transaction_schema(tx_body):
     try:
         jsonschema.validate(tx_body, TX_SCHEMA)
     except jsonschema.ValidationError as exc:
-        raise SchemaValidationError(str(exc))
+        raise SchemaValidationError(str(exc)) from exc
```

Comment:
I think so if we wish to keep the original message, but I am actually not completely sure.